pro/util.go

```diff
@@ -52,5 +51,13 @@ func getCurrentServerUsage() (limits Usage) {
 	if err == nil {
 		limits.Egresses = len(egresses)
 	}
+	relays, err := logic.GetRelays()
```

Comment:
move GetRelays to Pro pkg

Reply:
https://github.com/gravitl/netmaker/pull/2583/commits/682059c9482e5e7f9fa56c1e4f5c590cb2b8ac4e